src/BlazorUI/Bit.BlazorUI/Components/Modal/BitModal.scss

```diff
@@ -0,0 +1,104 @@
+﻿@import "../../Styles/functions.scss";
+
+.bit-mdl {
+    font-family: $typography-font-family;
+    font-size: spacing(1.75);
+    font-weight: 400;
+    z-index: $space-overlay-zindex0;
+    position: fixed;
+    inset: 0;
+
+    &.absolute {
+        position: absolute;
+        z-index: unset;
+    }
+
+    .modal {
+        position: absolute;
+        display: flex;
+        height: 100%;
+        width: 100%;
+        opacity: 1;
+        pointer-events: auto;
+        background-color: transparent;
+        transition: opacity 300ms ease 0s;
+    }
+
+    .overlay {
+        position: absolute;
+        z-index: 0;
+        inset: 0;
+        background-color: rgba(0, 0, 0, 0.4);
+    }
+
+    .scroll-content {
+        flex-grow: 1;
+        overflow-y: auto;
+        max-height: 100vh;
+        display: flex;
+        flex-direction: column;
+    }
+
+    .container {
+        box-shadow: rgba(0, 0, 0, 22%) 0 spacing(3.25) spacing(7.25) 0, rgba(0, 0, 0, 18%) 0 spacing(0.625) spacing(1.75) 0;
+        border-radius: spacing(0.25);
```

Comment:
Plz, use the `$shape-border-radius` variable instead of the `spacing` function

Reply:
not the same, we need to discuss it and probably add new variables